changelog

```diff
@@ -1,3 +1,14 @@
+turnkey-gitea-16.1 (1) turnkey; urgency=low
+
+  * Update Gitea to latest upstream (1.13.1).
```

Comment:
FWIW, I've been removing the brackets for these. I.e installed of this, I suggest: `Update Gitea to latest upstream: 1.13.1.`.